language/tools/move-package/src/resolution/dependency_graph.rs

```diff
@@ -42,6 +42,44 @@ use super::{
 ///
 /// In order to be `BuildConfig` agnostic, it contains `dev-dependencies` as well as `dependencies`
 /// and labels edges in the graph accordingly, as `DevOnly`, or `Always` dependencies.
+///
+///
```

Comment:
nit: just one line break between paragraphs?

Reply:
That was kind of intentional to separate to somewhat different parts of this large doc comment but I agree that it does not look that great